src/software/SfM/main_ComputeStructureFromKnownPoses.cpp

```diff
@@ -178,7 +178,7 @@ int main(int argc, char **argv)
     }
     // Compute the tracks from the pairwise estimation
     // Compute tracks from matches
-    const int min_track_length = 2;
+    const int min_track_length = 3;
```

Comment:
I will not change the default value.
But we can make it a configurable value in the command line.